dotfiles/ANSWERS.md

```diff
@@ -0,0 +1,3 @@
+I know my example dot files are pretty pathetic haha but I haven't done too much configuration recently to my environment. My main development machine used to be a macbook pro and I just used default git commands(no aliases), textmate and the terminal. Even on my Trisquel box now I just use defaults. When I worked in San Fran years ago for ModCloth I used to utilize config files more in that work environment but I just haven't had the need to really customize things. Sorry if that's a lame excuse. :) Also with my latest contract work, I've been stuck in Microsoft land so my life for the past 6+ months has been Visual Studio.
```

Comment:
I forgot to mention when I use Vim I like using the Janus distribution. It comes with lots of nice plugins like a file tree browser and easy file searching like Command-T.